x-pack/plugins/ml/server/models/data_recognizer/modules/siem_winlogbeat/manifest.json

```diff
@@ -1,100 +1,172 @@
 {
-  "id": "siem_winlogbeat",
-  "title": "Security: Winlogbeat",
-  "description": "Detect unusual processes and network activity in Winlogbeat data.",
-  "type": "Winlogbeat data",
+  "id": "siem_auditbeat",
```

Comment:
Did you mean to switch all these `winlogbeat`fields to refer to `auditbeat`?  The change to the module `id` is what is making the CI get_modules test fail.

Reply:
ah, yes this is my mistake. thank you! fixing.